test/conftest.py

```diff
@@ -0,0 +1,117 @@
+"""PyTest configuration module. Defines useful fixtures, command line args."""
+import logging
+import os
+from pathlib import Path
+
+import pudl
+import pytest
+import sqlalchemy as sa
+from pudl.output.pudltabl import PudlTabl
+
+import pudl_rmi
+
+logger = logging.getLogger(__name__)
+
+
+def pytest_addoption(parser):
+    """Add a command line option Requiring fresh data download."""
+    parser.addoption(
+        "--cached-plant-parts-eia",
+        action="store_true",
+        default=False,
+        help="Use the pickled EIA Plant Parts List.",
+    )
+    parser.addoption(
+        "--cached-deprish",
+        action="store_true",
+        default=False,
+        help="Use the pickled Depreciation data.",
+    )
+    parser.addoption(
+        "--cached-deprish-eia",
+        action="store_true",
+        default=False,
+        help="Use the pickled Depreciation to EIA connection.",
+    )
+    parser.addoption(
+        "--cached-ferc1-eia",
+        action="store_true",
+        default=False,
+        help="Use the pickled FERC 1 to EIA connection",
+    )
+
+
+@pytest.fixture(scope='session')
+def pudl_settings_fixture():  # noqa: C901
```

Comment:
this is not a this pr comment, but every time i see this function (as it looks like it is a direct copy paste from pudl's tests...) i think we need some way to import settings or some simpler way to configure this.

Reply:
Yes. I think we should turn `pudl_settings` into a Pydantic `Settings` model, which can have all of this logic inside it (and also refer to environment variables and other sources of settings-like information)